client/INLClient.py

```diff
@@ -46,10 +52,18 @@ def check_server(self, server):
         getter = JobGetter(self.client_info)
         claimed = getter.find_job()
         if claimed:
+            if self.get_client_info('manage_build_root'):
+                self.create_build_root()
+
             load_modules = settings.CONFIG_MODULES[claimed['config']]
             os.environ["CIVET_LOADED_MODULES"] = ' '.join(load_modules)
             self.modules.clear_and_load(load_modules)
             self.run_claimed_job(server[0], [ s[0] for s in settings.SERVERS ], claimed)
+            self.client_info["jobs_ran"] += 1
```

Comment:
What happens if "jobs_ran" doesn't exist?

Reply:
I think if you have a get_client_info, you might as well have a set_client_info and make self.client_info into self.__client_info to attempt to force use of the setter/getter

Reply:
Great minds think alike - done and done